backend/prompt_studio/prompt_studio_output_manager/views.py

```diff
@@ -49,3 +55,50 @@ def get_queryset(self) -> Optional[QuerySet]:
             queryset = PromptStudioOutputManager.objects.filter(**filter_args)
 
         return queryset
+
+    def get_output_for_tool_default(self, request: HttpRequest) -> Response:
+        # Get the tool_id from request parameters
+        # Get the tool_id from request parameters
+        tool_id = request.GET.get("tool_id")
+        document_manager_id = request.GET.get("document_manager")
+        tool_validation_message = PromptOutputManagerErrorMessage.TOOL_VALIDATION
+        if not tool_id:
+            raise APIException(detail=tool_validation_message, code=404)
```

Comment:
```suggestion
            raise APIException(detail=tool_validation_message, code=400)
```
My only concern over these messages is that we'd show a toast showing `tool_id parameter is required`. As a user what action do you think you'll be able to take here? But then again, if FE is the client - these seem valid

Reply:
I dont think user can do anything here but we need too_id from FE. so if we're not getting tool_id from FE then it will be more appropriate to say what is the error. please suggest a message If you think we can have more meaningful one.

Reply:
@chandrasekharan-zipstack This will be helpful in case of development in the frontend and if people try to hit the API directly with postman.

Anyways after proper integration in the UI, This particular error won't be displayed as toast because UI will always send the ID.